tests/layer_tests/mo_python_api_tests/test_mo_convert_tf.py

```diff
@@ -587,6 +587,27 @@ def __call__(self, input1):
     return model, model_ref, {'example_input': example_input}
 
 
+def create_keras_layer_with_string_tensor(tmp_dir):
+    import tensorflow as tf
+
+    x = tf.keras.Input(shape=[1], name="input", dtype=tf.dtypes.int64)
+    var = tf.Variable("Text_1")
+    const = tf.constant("Text_2")
+    hash = tf.strings.to_hash_bucket([const, var], 3)
+
+    model = tf.keras.Model(inputs=[x], outputs=[hash + x])
+    example_input = tf.constant(3)
+
+    param = ov.opset8.parameter([], dtype=np.int32)
+    conv = ov.opset8.convert(param, np.int64)
+    const = ov.opset8.constant([0, 2], dtype=np.int64)
```

Comment:
I am not sure that we test the required logic. I propose to create a graph with two sub-graphs: 1. Add(param1_float, param2_float) 2. Const(string)

We have transformation removing Const->Result this way the resulted graph will be with Add operation and two flow params

Reply:
Added test with such graph.